src/logic/function.lean

```diff
@@ -44,6 +44,12 @@ mt (assume h, hf h)
 def injective.decidable_eq [decidable_eq β] (I : injective f) : decidable_eq α
 | a b := decidable_of_iff _ I.eq_iff
 
+def injective.of_comp {γ : Sort w} {g : γ → α} (I : injective (f ∘ g)) : injective g :=
```

Comment:
@urkud It might be a good idea to run `#sanity_check` at the bottom of these long new files.
@cipher1024 Should we add a sanity check stage to Travis? I think we shouldn't do that before there is some `insane` tag to tell sanity check that things are fine.
And even then, maybe we only want to catch `def`s that should be `lemma`s. The other checks might depend too much on human insight/taste.

Reply:
@rwbarton @jcommelin #sanity_check done.